lib/Tvm.js

```diff
@@ -154,11 +237,23 @@ class Tvm {
    */
   async processRequest (params) {
     try {
+      // 0. validate Adobe I/O Api Gateway service token
+      if (params.disableAdobeIOApiGwTokenValidation !== 'true') {
```

Comment:
Can we have this flag passed as boolean rather than string?

Reply:
Not sure if can do this in the manifest default params, for now everything comes in as a string

Reply:
actually in the wskdeploy specs there is a type parameter https://github.com/apache/openwhisk-wskdeploy/blob/master/specification/html/spec_parameters.md#fields, checking if we support this in our tooling..

Reply:
so I did some testing and params type are correctly parsed and set in `params` in those cases:
```yaml
inputs:
  a: 123 # integer
  b: true # boolean
  c: 'abc' # string
```

However the problem is that `disableAdobeIOApiGwTokenValidation` is set through an env variable `DISABLE_ADOBE_IO_API_GW_TOKEN_VALIDATION` and that environment variables always evaluate to strings.

And I also tried the following yaml syntax
```yaml
inputs:
  a: 
    type: integer
    value: $A_ENV # A_ENV=123
```
The deployed action gets `params.a='$A_ENV'`

So for now  we will have to continue with strings I guess